.github/instructions/dsc-community-style-guidelines.instructions.md

```
@@ -10,10 +10,11 @@ applyTo: "**"
 - **Function**: Private function
 - **Resource**: DSC class-based resource
 
-## Build & Test Workflow
-- Run in PowerShell, from repository root
-- Build before running tests: `.\build.ps1 -Tasks build`
-- Always run tests in new PowerShell session: `Invoke-Pester -Path @({test paths}) -Output Detailed`
+## Build & Test Workflow Requirements
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)